packages/frontend-2/components/viewer/saved-views/panel/views/Group.vue

```diff
@@ -50,13 +55,6 @@
       </div>
     </template>
   </LayoutDisclosure>
-  <ViewerSavedViewsPanelViewsGroupInner
-    v-else
```

Comment:
IMO u should've just wrapped the entire component w/ a new div that u add all of the dragging handlers on. 
if there's no longer this v-else, how are we ensuring auto-open for the ungrouped group?